lib/plugins/inspec-reporter-cli/README.md

```diff
@@ -0,0 +1,12 @@
+# cli reporter
```

Comment:
Looks good, do these sync up with our public `www` documentation? If so, do we need docs review or is this small enough not to need it?

Reply:
This *should* be simply an internal refactor - no change to the behavior of Chef InSpec and no change to www docs. I simply added a README because there is usually a README in each plugin directory.